src/common/plugin/types.ts

```diff
@@ -56,6 +57,11 @@ export interface EngineInfo {
   scriptTemplates?: ScriptTemplates
   // Codec Cleaners
   asBlockbookAddress?: Cleaner<string>
+  // Coin specific transaction handling
+  txSpecificHandling?: (
```

Comment:
Alternatively, you can make the `specialTx: TransactionSpecificResponse` an `unknown` field.